package.json

```diff
@@ -85,14 +86,13 @@
     "ejs": "~2.5.7",
     "enzyme": "~3.3.0",
     "enzyme-adapter-react-16": "~1.2.0",
-    "eslint": "~3.9.1",
-    "eslint-config-airbnb": "~13.0.0",
+    "eslint": "~5.4.0",
+    "eslint-config-airbnb": "~17.1.0",
     "eslint-config-standard": "~5.3.1",
```

Comment:
Should this be removed along with the plugin?

Reply:
Weird, I thought I had. But yes